### PR TITLE
[config][android] Handle remote config error

### PR DESCRIPTION
**Pull request by Ehesp**
https://github.com/invertase/react-native-firebase/issues/1288

### CODE REVIEW

File: android/src/main/java/io/invertase/firebase/config/RNFirebaseRemoteConfig.java [hunk 1]
```diff
@@ -119,9 +120,13 @@ private void fetchInternal(final Promise promise, Boolean withExpiration, long e
         @Override
         public void onComplete(@NonNull Task<Void> task) {
           if (task.isSuccessful()) {
-            promise.resolve("remoteConfigFetchStatusSuccess");
+            promise.resolve("config/success");
```
Comment (Salakar) [inline android/src/main/java/io/invertase/firebase/config/RNFirebaseRemoteConfig.java]: Could change to null - doesn't need to resolve any sort of meaningful value really.

File: android/src/main/java/io/invertase/firebase/config/RNFirebaseRemoteConfig.java [hunk 2]
```diff
@@ -119,9 +120,13 @@ private void fetchInternal(final Promise promise, Boolean withExpiration, long e
         @Override
         public void onComplete(@NonNull Task<Void> task) {
           if (task.isSuccessful()) {
-            promise.resolve("remoteConfigFetchStatusSuccess");
```
Comment (Salakar) [inline android/src/main/java/io/invertase/firebase/config/RNFirebaseRemoteConfig.java]: Could change to null - doesn't need to resolve any sort of meaningful value really.

File: ios/RNFirebase/config/RNFirebaseRemoteConfig.m [hunk 3]
```diff
@@ -49,7 +59,7 @@ @implementation RNFirebaseRemoteConfig
             (RCTPromiseRejectBlock) reject) {
     [[FIRRemoteConfig remoteConfig] fetchWithCompletionHandler:^(FIRRemoteConfigFetchStatus status, NSError *__nullable error) {
         if (error) {
-            reject(convertFIRRemoteConfigFetchStatusToNSString(status), error.localizedDescription, error);
+            reject(convertFIRRemoteConfigFetchStatusToNSString(status), convertFIRRemoteConfigFetchStatusToNSStringDescription(status), error);
         } else {
             resolve(convertFIRRemoteConfigFetchStatusToNSString(status));
```
Comment (Salakar) [inline ios/RNFirebase/config/RNFirebaseRemoteConfig.m]: Could change to nil - doesn't need to resolve any sort of meaningful value really.

File: ios/RNFirebase/config/RNFirebaseRemoteConfig.m [hunk 4]
```diff
@@ -63,7 +73,7 @@ @implementation RNFirebaseRemoteConfig
         rejecter:(RCTPromiseRejectBlock)reject) {
     [[FIRRemoteConfig remoteConfig] fetchWithExpirationDuration:expirationDuration.doubleValue completionHandler:^(FIRRemoteConfigFetchStatus status, NSError *__nullable error) {
         if (error) {
-            reject(convertFIRRemoteConfigFetchStatusToNSString(status), error.localizedDescription, error);
+            reject(convertFIRRemoteConfigFetchStatusToNSString(status), convertFIRRemoteConfigFetchStatusToNSStringDescription(status), error);
         } else {
             resolve(convertFIRRemoteConfigFetchStatusToNSString(status));
```
Comment (Salakar) [inline ios/RNFirebase/config/RNFirebaseRemoteConfig.m]: Could change to nil - doesn't need to resolve any sort of meaningful value really.

**Comment by Ehesp** [comment]
@Salakar sorted them changes

**Pull request merged**